torch/_torch_docs.py

```diff
@@ -2586,7 +2586,7 @@ def parse_kwargs(desc):
 Args:
     input (Tensor): the input tensor
     dim (int): the dimension to reduce
-    keepdim (bool): whether the output tensors have :attr:`dim` retained or not
+    keepdim (bool, optional): whether the output tensors have :attr:`dim` retained or not
```

Comment:
nit: Note the default "default: False"

Reply:
Yes. The default value of keepdim is False. [Line 2575](https://github.com/pytorch/pytorch/pull/15068/files#diff-cfb065e43691d37ba30f3856fe4352b3L2575)

Reply:
Yes, it would be nice if we noted the default for keepdim here in the documentation.